src/base/utils/fs.cpp

```diff
@@ -518,3 +518,10 @@ QString Utils::Fs::cacheLocation()
         locationDir.mkpath(locationDir.absolutePath());
     return location;
 }
+
+QString Utils::Fs::tempPath()
+{
+    static const QString path = QDir::tempPath() + "/.qBittorrent/";
+    QDir().mkdir(path);
```

Comment:
Originally I mean when it's still running, but I guess the concept applies to both, can't see the difference (as long as the temp folder will exist).